tests/run_example.rs

```diff
@@ -0,0 +1,39 @@
+#[test]
+fn run_example() {
+    let output = std::process::Command::new("cargo")
+        .args(["nextest", "run", "--test=example", "--color=never"])
```

Comment:
Yeah, definitely want to force color off in this test. (Again, scraping human-readable output isn't great, but hopefully we have a better story soon.)